@bull-board/api@3.1.0 typings error

Hello,

First of all thank you for what you have implemented so far on this project 👍

I encountered the following errors in CI environment:

```bash
node_modules/@bull-board/api/typings/app.d.ts:1:29 - error TS2307: Cannot find module '../src/queueAdapters/base' or its corresponding type declarations.
1 import { BaseAdapter } from '../src/queueAdapters/base';
                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@bull-board/api/typings/app.d.ts:2:26 - error TS2307: Cannot find module '../src/constants/statuses' or its corresponding type declarations.
2 import { STATUSES } from '../src/constants/statuses';
```

It looks like the imports of the typings are referencing the source code instead of the built one, or am I missing something ? 🤔 

Thanks in advance for your insights !

--- a/packages/api/typings/app.d.ts
+++ b/packages/api/typings/app.d.ts
@@ -1,5 +1,5 @@
-import { BaseAdapter } from '../src/queueAdapters/base';
-import { STATUSES } from '../src/constants/statuses';
+import { BaseAdapter } from '../dist/queueAdapters/base';
+import { STATUSES } from '../dist/constants/statuses';
 
 export type JobCleanStatus = 'completed' | 'wait' | 'active' | 'delayed' | 'failed';
 

